src/main/java/com/xiaomi/infra/pegasus/client/PegasusClientInterface.java

```diff
@@ -44,6 +44,9 @@
   /**
    * Open a table, and prepare the sessions and route-table to the replica-servers.
    *
+   * <p>Note: this interface is deprecated, retaining it only for compatibility, please see {@link
+   * PegasusClientInterface#openTable(String, TableOptions)}
+   *
    * <p>Please notice that pegasus support two kinds of API: 1. the client-interface way, which is
```

Comment:
Remove this repeated paragraph.

It'd be better to tell the users what deprecation of this interface means. See hbase-client 
https://github.com/apache/hbase/blob/master/hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncTable.java#L239:

```
   * @deprecated Since 3.0.0, will be removed in 4.0.0. For internal test use only, do not use it
   *   any more.
```